airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -144,6 +144,9 @@ definitions:
         type: array
         items:
           type: string
+      namespace:
```

Comment:
can you move it close to name since these two work together?

Reply:
I wonder if we should create an object that contains both name & namespace that we can use for both the stream and the configured stream. 

WDYT?
```
StreamName {
   string: name
   string: namespace
}
```